integration-tests/cucumber-tests-platform-distributionautomation/src/test/java/org/opensmartgridplatform/cucumber/platform/distributionautomation/glue/steps/database/ws/DistributionAutomationNotificationWebServiceConfigurationRepository.java

```diff
@@ -12,6 +12,6 @@
 import org.opensmartgridplatform.adapter.ws.domain.repositories.NotificationWebServiceConfigurationRepository;
 import org.springframework.stereotype.Repository;
 
-@Repository
+@Repository("wsDistributionAutomationNotificationWebServiceConfigurationRepository")
```

Comment:
Heb je overwogen om de naam van de class WsDistributionAutomationNotificationWebServiceConfigurationRepository te maken (dit geldt voor meerdere classes in dit package)